docs/user-guide/timepix_images.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Timepix Images (Fe Sample and Open Beam)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ess.imaging.data import get_fe_timepix_image_path, get_ob_timepix_image_path\n",
+    "from ess.imaging.io import load_tiff\n",
+    "\n",
+    "sample_image_path = get_fe_timepix_image_path()\n",
+    "openbeam_image_path = get_ob_timepix_image_path()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define Workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib\n",
+    "\n",
+    "import scipp as sc\n",
+    "import sciline as sl\n",
+    "\n",
+    "from typing import NewType, TypeVar\n",
+    "\n",
+    "Sample = NewType('Sample', int)\n",
+    "OpenBeam = NewType('OpenBeam', int)\n",
+    "RunType = TypeVar('RunType', Sample, OpenBeam)\n",
+    "\n",
+    "\n",
+    "class FilePath(sl.Scope[RunType, pathlib.Path], pathlib.Path): ...\n",
+    "\n",
+    "\n",
+    "class TiffImage(sl.Scope[RunType, sc.DataArray], sc.DataArray): ...\n",
+    "\n",
+    "\n",
+    "NormalizedImage = NewType('NormalizedImage', sc.DataArray)\n",
+    "\n",
+    "\n",
+    "def load_image(path: FilePath[RunType]) -> TiffImage[RunType]:\n",
+    "    easier_dims = {'dim_0': 'x', 'dim_1': 'y', 'dim_2': 't'}\n",
```

Comment:
Shouldn't adding the coords be done inside the image loader?

Reply:
I don't know... tiff image can't really carry any coordinates.
Should we just make a dimensionless coordinate to start with?